docs/source/gs-processing/usage/example.rst

```diff
@@ -166,18 +168,14 @@ to process the data and create the output on our local storage.
 We will provide an input and output prefix for our data, passing
 local paths to the script.
 
-We also provide the argument ``--num-output-files`` that instructs PySpark
-to try and create output with 4 partitions [#f1]_.
-
 Assuming our working directory is ``graphstorm/graphstorm-processing/``
 we can use the following command to run the processing job locally:
 
 .. code-block:: bash
 
     gs-processing --config-filename gconstruct-config.json \
         --input-prefix ./tests/resources/small_heterogeneous_graph \
-        --output-prefix /tmp/gsprocessing-example/ \
-        --num-output-files 4
+        --output-prefix /tmp/gsprocessing-example/
```

Comment:
Are we removing ``--num-output-files``?

Reply:
Yes, better to have the default value of `-1` that will ensure all cluster resources are used. Otherwise we run the risk of users copy pasting this snippet and only using 4 cores in any size cluster.